src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Circle extends Color {
+    private double radius;
+
+    public Circle(String color, double radius) {
```

Comment:
The constructor should not accept a color string directly if Circle is not extending Color. Instead, consider having a Color field in the Circle class and initialize it in the constructor.